alignak/comment.py

```diff
@@ -109,9 +121,19 @@ def __init__(self, params):
         :type expire_time: int
         :return: None
         """
-        super(Comment, self).__init__(params)
-        self.entry_time = int(time.time())
-        self.fill_default()
+        if 'uuid' not in params:
+            super(Comment, self).__init__(params)
```

Comment:
Comment is not parsed (no configuration!) and it is serialized specifically for the retention ... this is why the __init__ is not using parsing and only the uuid can be used to know if we create or unserialize ... two uuid test, because we need to initializa, restore params and restore entry_time if it exists. No other solution to me for this ...